app/helpers/hyrax/collections_helper.rb

```diff
@@ -43,6 +43,29 @@ def collection_type_label(collection_type_gid)
       collection_type_gid
     end
 
+    def banner_file(collection_id)
+      # Find Banner filename
+      ci = CollectionBrandingInfo.where(collection_id: collection_id.to_s).where(role: "banner")
+      # @banner_file = File.split(ci[0].local_path).last unless ci.empty?
+      # @banner_file_location = ci[0].local_path unless ci.empty?
```

Comment:
cruft?

Reply:
Again, I didn't touch this file so I don't know. I can ask Lynette who did if you think it would be helpful.